arcor2_AREditor/Assets/BASE/Scripts/GameManager.cs

```diff
@@ -870,7 +870,7 @@ public enum EditorStateEnum {
 
                     SystemInfo = systemInfo;
                     ServerVersion.text = "Editor version: " + Application.version +
-                        "\nServer version: " + systemInfo.Version;
+                        "\nServer version: " + systemInfo._Version;
```

Comment:
Do you have an idea why this changed from `Version` to `_Version`?

Reply:
I don't know why, after generating the models, I had to change it to `_Version`, otherwise it would not compile.

Reply:
Strange. Hopefully, nothing will break :-)